Rakefile

```diff
@@ -277,6 +286,7 @@ namespace :install do
         code_studio_build = CDO.use_my_code_studio ? code_studio_dir('build') : 'code-studio-package'
         RakeUtils.ln_s code_studio_build, dashboard_dir('public','code-studio')
       end
+      ensure_code_studio_package
```

Comment:
Ensuring the code-studio package feels more like an install step for dashboard, not for code-studio itself.  Maybe even a build step for dashboard.